### PR TITLE
Feature/1 upgrade dotnet6

### PR DESCRIPTION
**Pull request by TheTechArch**

### CODE REVIEW

File: src/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/sdk:5.0-alpine AS build
+FROM mcr.microsoft.com/dotnet/sdk:6.0.102-alpine3.14 AS build
```
Comment (tjololo) [inline src/Dockerfile:1]: Is there a reason why we are pinning the alpine version now?

File: src/Dockerfile [hunk 2]
```diff
@@ -8,7 +8,7 @@ WORKDIR /App
 RUN dotnet build App.csproj --configuration Release --output /app_output
 RUN dotnet publish App.csproj --configuration Release --output /app_output
 
-FROM mcr.microsoft.com/dotnet/aspnet:5.0-alpine AS final
+FROM mcr.microsoft.com/dotnet/aspnet:6.0.2-alpine3.14 AS final
```
Comment (tjololo) [inline src/Dockerfile:11]: Same as for the build container, why are we pinning the alpine version?

File: src/App.IntegrationTests/App.IntegrationTestsRef.csproj [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="FluentAssertions" Version="6.4.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="5.0.14" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
+    <PackageReference Include="coverlet.collector" Version="3.1.2">
```
Comment (tjololo) [inline src/App.IntegrationTests/App.IntegrationTestsRef.csproj:9]: nice! Love the alphabetical order

**Review by tjololo — COMMENTED** [review]
Looks good, only one question

File: src/Dockerfile (hunk 1 above)
Comment (RonnyB71) [reply src/Dockerfile:1]: Good question, don't think so. @TheTechArch did you experience issues when you did this?

Comment (RonnyB71) [reply src/Dockerfile]: Changed to use latest.

File: src/Dockerfile (hunk 2 above)
Comment (RonnyB71) [reply src/Dockerfile]: Changed to use latest.

File: src/App.IntegrationTests/App.IntegrationTestsRef.csproj (hunk 3 above)
Comment (RonnyB71) [reply src/App.IntegrationTests/App.IntegrationTestsRef.csproj:9]: Should be fixed:)

**Review by tjololo — APPROVED** [review]
Looks good to me :godmode:

**Pull request merged**